src/components/chip/chip.scss

```diff
@@ -0,0 +1,69 @@
+@use '../../style/mixins';
+
+/**
+* @prop --chip-max-width: Maximum width of the chip. Defaults to `10rem`. Keep in mind that the chips should not appear too big.
+*/
+
+:host(limel-chip) {
+    --limel-chip-height: 1.75rem;
```

Comment:
💡 _this is an internal variable, starting with `--limel-` prefix!_

The current `mdc-chip` is `32px` tall. Also, `limel-button` is `36px` tall. I intentionally want the new chips to be `28px` tall! 

**Why?**

1. I don't want consumers to use chips instead of buttons, or put them beside a button in the UI.
2. I want to make all input fields smaller in height. All should become `36px`, like `limel-button`. This means, in the chip-set `type="input"`, we will need more space around the chips. And this will give us that height.
     
    However, before resizing all input fields, we can use this internal CSS var to render the chips larger in the chip-set.